new/rfcXXX-ruby_data_bag_items.md

```diff
@@ -0,0 +1,131 @@
+---
+RFC: unassigned
+Author: Eric Krupnik <ekrupnik@copyright.com>
+Status: Draft
+Type: Standards Track
+---
+# Ruby data bag items
+
+Currently, Chef only supports JSON data bag items, and not Ruby. Chef **does**
+support having other configuration-based files (such as `environments` and `roles`)
+be written in either JSON or Ruby. JSON does not allow comments, so it becomes
```

Comment:
@lamont-granquist Can you please point me at the documentation (or an example) of a Chef data bag item (JSON) which has comments. I have not been able to find anything online regarding data bag items supporting comments. Because by default, I do not believe that JSON supports comments.